modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/DLReferencesExportImportContentProcessor.java

```diff
@@ -347,9 +347,26 @@ private String _getUuid(String s) {
 		Matcher matcher = _uuidPattern.matcher(s);
 
 		String uuid = StringPool.BLANK;
+		String separator = StringPool.BLANK;
 
 		while (matcher.find()) {
-			uuid = matcher.group(0);
+			Integer lastMatchIndex =
+				s.indexOf(matcher.group(0)) +
+					matcher.group(
+						0
+					).length();
+
+			if (lastMatchIndex < s.length()) {
+				separator = s.substring(lastMatchIndex, lastMatchIndex + 1);
+			}
+
+			if (Validator.isNull(separator) ||
+				(Validator.isNotNull(separator) &&
+				 (separator.equals(StringPool.QUESTION) ||
+				  separator.equals(StringPool.AMPERSAND)))) {
+
+				uuid = matcher.group(0);
+			}
```

Comment:
Instead of this piece of code, why not use positive lookahead? Take a look at https://www.regular-expressions.info/lookaround.html. Simply adding `(?=[?&])` to the regexp should work and is much simpler.

Reply:
I made the requested change and force pushed it.

Thank you,